### DISCUSSION TITLE
Categories Report in Multi-Currency

### DISCUSSION THREAD

**Discussion by R-Nabil** [firefly-iii/firefly-iii#9202] [General]
Hello,

Is there a way to generate the Report for Categories with a single "base" currency ? Now few categories are duplicated, which makes it less useful to understand the spending related to a given category.

It could be interesting if this was optionnal, with way of changing this base currency as well

**Comment by JC5** [comment]
Hi, great question!

There are several issues and questions where I explain this is already on my list :)

**Reply by R-Nabil** [reply]
I've done some research and couldn't find anything.

**Reply by R-Nabil** [reply]
Hey. Sorry again but I can't find that list you mentioned ?
